client/src/components/utils/Footer/style.css

```diff
@@ -0,0 +1,93 @@
+.footer {
```

Comment:
use a CSS formatter or prettier to formate the CSS and to remove unnecessary empty whitespaces